backend/eventloop/application_event_loop/application_event_runner_deployments.go

```diff
@@ -248,6 +249,11 @@ func (a applicationEventLoopRunner_Action) handleNewGitOpsDeplEvent(ctx context.
 		userError := "Unable to reconcile the ManagedEnvironment. Verify that the ManagedEnvironment and Secret are correctly defined, and have valid credentials"
 		devError := fmt.Errorf("unable to get or create managed environment, isworkspacetarget:%v: %v", isWorkspaceTarget, err)
 
+		// If we recognize this error is a connection error due to the user providing us invalid credentials, don't bother to log it.
```

Comment:
I think this will not only block the log, but also block the update of the `GitOpsDeployment`'s .status field, which is where the error is communicated back to the user? We definitely still want to log the error to the .status field of the GitOpsDeployment

Reply:
Yes it will, I have changed it and now error is ignored after status is updated here:
https://github.com/redhat-appstudio/managed-gitops/pull/635/files#diff-08e1558d59743d0033fba6e63425c93a27f78f971a4dca44b4bd386ae3214f4cR392
PTAL.